main/src/ca/uwaterloo/flix/language/phase/jvm/GenContinuationInterfaces.scala

```diff
@@ -64,8 +66,28 @@ object GenContinuationInterfaces {
     // We can use `getErasedType` to map it down into one of the primitive types or to Object.
     //
 
+    // Class visitor
+    val visitor = new ClassWriter(ClassWriter.COMPUTE_FRAMES){
```

Comment:
Looks like the same class writer.

I think I will add a helper method and put it into a separate object, `AsmOps` where we can keep things specifically for working with Asm